.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
```

Comment:
Why did you move the husky tasks from the `package.json` to their own shell scripts? I personally like them in the `package.json` since it keeps the system configuration altogether. This also might cause issues with windows. Your thoughts?

Reply:
@kpfromer - I updated husky to version 7 and I'm not sure at what version, but Husky no longer supports config in package.json. 

https://blog.typicode.com/husky-git-hooks-javascript-config/

Scott

Reply:
Dang, I really liked the `package.json` configuration. Thanks for the details.